depends/packages/rapidcheck.mk

```diff
@@ -1,11 +1,11 @@
 package=rapidcheck
-$(package)_version=3eb9b4ff69f4ff2d9932e8f852c2b2a61d7c20d3
+$(package)_version=d9482c683429fe79122e3dcab14c9655874aeb8e
 $(package)_download_path=https://github.com/emil-e/rapidcheck/archive
 $(package)_file_name=$($(package)_version).tar.gz
 $(package)_sha256_hash=5fbf82755c9a647127e62563be079448ff8b1db9ca80a52a673dd9a88fdb714b
 
 define $(package)_config_cmds
-  cmake -DCMAKE_INSTALL_PREFIX=$($(package)_staging_dir)$(host_prefix) -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=true -DRC_INSTALL_ALL_EXTRAS=ON
+  cmake -DCMAKE_INSTALL_PREFIX=$($(package)_staging_dir)$(host_prefix) -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=true -DRC_ENABLE_BOOST_TEST=ON
```

Comment:
ACK [`DRC_ENABLE_BOOST_TEST`](https://github.com/emil-e/rapidcheck/blob/master/extras/CMakeLists.txt#L25). This seems better than [`DRC_INSTALL_ALL_EXTRAS`](https://github.com/emil-e/rapidcheck/blob/master/extras/CMakeLists.txt#L20).